src/timeInput.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import TimeInput from './TimeInput';
+
+describe('TimeInput', function() {
+
+	let timeInputComponent,
+		timeInputEl;
+
+	const timeValue = '22:00',
+		newTime = '23:00',
+		callbackSpy = jasmine.createSpy('foo');
```

Comment:
jasmine? do you mean jest?  alternative writing of this would be
`callbackSpy = jest.fn()`